drivers/char/rust_example.rs

```diff
@@ -136,6 +136,8 @@ impl KernelModule for RustExample {
 
         let mut chrdev_reg =
             chrdev::Registration::new_pinned(cstr!("rust_chrdev"), 0, &THIS_MODULE)?;
+        // Register the same kind of device twice, we're just demonstrating
+        // that you can use multiple minors.
```

Comment:
nit: Perhaps say something about chrdev_reg allowing 2 registrations because its type is `chrdev::Registration<2>`?

Reply:
good call